README.md

```diff
@@ -112,9 +112,9 @@ If a given part isn't set, it'll be excluded.
 
 ## Debugging
 
-You can start the jmx's scraper in standalone mode in order to debug what is called 
+You can start the jmx's scraper in standalone mode (after building the packages with Maven) in order to debug what is called 
 
-`java -cp jmx_exporter.jar io.prometheus.jmx.JmxScraper  service:jmx:rmi:your_url`
+`java -cp jmx_exporter/collector/target/collector*.jar  io.prometheus.jmx.JmxScraper  service:jmx:rmi:your_url`
```

Comment:
This path doesn't seem right, wouldn't you already be in the jmx_exporter directory?

Reply:
Well spotted. I've pushed another change to make the entire thing explicit.